_docs/setup/kubernetes/manual-sidecar-inject.md

```diff
@@ -181,17 +201,23 @@ applications which are redirected to proxy.
           apiVersion: v1
           fieldPath: metadata.namespace
     -
-      name: POD_IP
+      name: INSTANCE_IP
```

Comment:
in the future we can rename this to POD_IP, it seems to be an unwritten standard.

Reply:
Changed to reflect the actual value used [here](https://github.com/istio/pilot/blob/master/platform/kube/inject/inject.go#L426)